### ISSUE TITLE
Demographic inference methods to apply

### ISSUE THREAD

**Issue by arundurvasula** [popgensims/analysis#1]
We talked about gathering a list of methods to apply to the simulations on the call today

# squiggly line

- MSMC
- stairway plot
- SMC++

# multipopulation

- dadi
- fastsimcoal
- momi
- GPhocs

Please comment on additions! Not sure where we should keep the final list.

**Comment by andrewkern** [comment]
Question-- should we organize analysis by method or by organism?

**Comment by jeromekelleher** [comment]
> Question-- should we organize analysis by method or by organism?

What about by "thing-we're-trying-to-estimate"? That would map well into sections for the paper then.

**Comment by petrelharp** [comment]
Not sure: how much organism- and method-specific code is there? We probably have to do an example or two to see.

**Comment by andrewkern** [comment]
check out what I sketched up [here](https://github.com/andrewkern/analysis)

**Comment by petrelharp** [comment]
It was proposed in the meeting just now to add msmc to the multiple population list also.

**Comment by andrewkern** [comment]
yes and Kirk and company have volunteered to fun fastsimcoal2 for us. Thanks guys!
